lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -175,6 +175,16 @@ const FeatureInfoSection = createReactClass({
 
     render() {
         const catalogItemName = (this.props.catalogItem && this.props.catalogItem.name) || '';
+        let baseFilename = catalogItemName;
+        if (this.props.position) {
+            const position = Ellipsoid.WGS84.cartesianToCartographic(this.props.position);
+            const latitude = CesiumMath.toDegrees(position.latitude).toFixed(5);
+            const longitude = CesiumMath.toDegrees(position.longitude).toFixed(5);
+            // Check that the catalogItemName doesn't already contain the lat lon with the same or more precision.
+            if ((typeof baseFilename !== 'string') || (baseFilename.indexOf(latitude) === -1) || (baseFilename.indexOf(longitude) === -1)) {
```

Comment:
Well spotted, thanks for pointing this out. 

Have a look at the latest change and let me know what you think.

I came up with two alternate solutions for this , either checking for both rounded up and down values or could have had the check perform just to number of decimal places -1. Neither are ideal, because checking for both rounding seems overly complex for the situation and reducing the precision with which we check and using floor means that the check vs the result and effect is somewhat imbalanced. Have gone with the more complex solution for now.